src/app/fyle/my-expenses/my-expenses.page.ts

```
@@ -569,14 +571,13 @@ export class MyExpensesPage implements OnInit {
-        const queryParams = params.queryParams || {};
+        let queryParams = params.queryParams || {};
 
         queryParams.report_id = queryParams.report_id || 'is.null';
         queryParams.state = 'in.(COMPLETE,DRAFT)';
 
         if (params.searchString) {
-          queryParams.q = params.searchString;
-          queryParams.q = queryParams.q + ':*';
+          queryParams = this.apiV2Service.extendQueryParamsForTextSearch(queryParams, params.searchString);
         } else if (params.searchString === '') {
           delete queryParams.q;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)